angularjs-1.5-basics/src/app/landing-page/common/renderComponent.directive.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+
+var renderComponent = function () {
+    return {
+        restrict: 'A',
+        link: function (scope, element, attrs, controllers) {
+            return element.append(attrs.component);
+        }
+    };
+
+};
+
+angular.module('landing-page')
```

Comment:
__[jshint]__
* 'angular' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->